grc.fraudDetection_web/src/main/java/com/typedb/examples/fraud/util/Formatter.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (C) 2022 Vaticle
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.typedb.examples.fraud.util;
+
+public class Formatter {
+
+  public static String formatDateTime(String dateTime) {
```

Comment:
It's a little suspicious to me that we're taking an argument called `dateTime` as a string when a native DateTime class exists within Java, as well as the fact that we're correctly storing timestamps as `datetime` in our schema. 

Returning a string is fine if we're using this function as the last step before substituting in a value to be submitted as a query, though maybe this is an indicator we should be using the query builder (e.g. `var("p").isa("person")`)

Reply:
The CSV file has a space between date and time, so we're just replacing it with a 'T' for TypeDB. The focus in this example is more on understanding TypeDB queries rather than using the builder API.

Reply:
Understood!